build.gradle

```diff
@@ -15,28 +15,36 @@ buildscript {
     repositories {
         mavenCentral()
         jcenter()
-        maven{
+        maven {
             url "https://plugins.gradle.org/m2/"
         }
+        maven {
+            url "https://maven.ej-technologies.com/repository"
+        }
     }
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "edu.sc.seis.gradle:launch4j:2.4.4"
+// commenting out the jfx plugin for now since we are bundling the Oracle JRE in dev
+//        classpath group: 'de.dynamicfiles.projects.gradle.plugins', name: 'javafx-gradle-plugin', version: '8.8.2'
         classpath group: 'org.jooq', name: 'jooq-meta-extensions', version: '3.11.2'
         classpath 'org.jooq:jooq-codegen:3.11.2'
         classpath 'org.xerial:sqlite-jdbc:3.25.2'
+        classpath group: 'com.install4j', name: 'install4j-gradle', version: '7.0.8'
     }
 }
 
 plugins {
     id 'java'
+    id 'java-library'
     id 'nu.studer.jooq' version '3.0.1'
+    id 'application'
 }
 
 group 'org.wycliffeassociates'
 apply plugin: 'kotlin'
 apply plugin: 'kotlin-kapt'
-apply plugin: 'edu.sc.seis.launch4j'
+apply plugin: 'install4j'
+//apply plugin: 'javafx-gradle-plugin'
```

Comment:
do we no longer need the javafx plugin?

Reply:
Not that I can tell since we are using the Oracle JRE.